Source/GRChomboCore/ChomboParameters.hpp

```diff
@@ -91,6 +89,14 @@ class ChomboParameters
             max_N = max(N, max_N);
         }
         coarsest_dx = L / max_N;
+      
+        pp.load("N1", N1);                                                                                                     
+        pp.load("N2", N2);                                                                                                     
+        pp.load("N3", N3);                                                                                                     
+                                                                                                                               
+        pp.load("center", center,                                                                                              
+                {0.5 * N1 * coarsest_dx, 0.5 * N2 * coarsest_dx, 
+                 0.5 * N3 * coarsest_dx}); // default to center
```

Comment:
Yes, I agree with @KAClough that these parameters shouldn't be reloaded as they are already loaded into `ivN` (although this is `N - 1` rather than `N` because this is the way Chombo wants it).
```suggestion
        IntVect Nvec = ivN + IntVect::Unit;                                                                                                               
        pp.load("center", center,                                                                                              
                {0.5 * Nvec[0] * coarsest_dx, 0.5 * Nvec[1] * coarsest_dx, 
                 0.5 * Nvec[2] * coarsest_dx}); // default to center
```